pkg/registry/service/strategy.go

```diff
@@ -90,7 +90,7 @@ func (svcStrategy) Export(obj runtime.Object, exact bool) error {
 		return nil
 	}
 	if t.Spec.ClusterIP != api.ClusterIPNone {
-		t.Spec.ClusterIP = ""
+		t.Spec.ClusterIP = "<unknown>"
```

Comment:
This prevents the exported resource from being created on another cluster, which defeats the purpose of export.